### PR TITLE
Fix no action multiple item selection message in project organizer

### PR DESCRIPTION
**Pull request by caneruguz**
## Purpose
Fix an issue introduced in a prior merge where the wrong template names were used to show multiple folder selection. 
Trello card: https://trello.com/c/v0aBR1cf

## Change
Correct the no-action grid template reference in project organizer

## Side effects
None.

### CODE REVIEW

**Comment by sloria** [comment]
@caneruguz Your fix is correct, but it still does not fix the issue in the Trello card. The problem is that the actions bar says "Select a row..." when multiple rows are already selected.

osf___dashboard

**Comment by caneruguz** [comment]
I think this last commit I made should resolve this issue. It was showing the blank template for non valid multiselections but now if there is multiselection and no action it will always show how many and say there are no actions.

File: website/static/js/projectorganizer.js [hunk 1]
```diff
@@ -782,7 +783,13 @@ function _poMultiselect(event, tree) {
                     return false;
                 });
             } else {
-                createBlankProjectDetail();
+                detailTemplateContext = {
+                    itemsCount: selectedRows.length
+                };
+                var theParentNode = selectedRows[0].parent();
```
Comment (sloria) [inline website/static/js/projectorganizer.js]: This doesn't appear to be used.

File: website/static/js/projectorganizer.js [hunk 2]
```diff
@@ -782,7 +783,13 @@ function _poMultiselect(event, tree) {
...
+                var displayHTML = multiItemDetailTemplateNoAction(detailTemplateContext);
+                $('.project-details').html(displayHTML);
+                $('.project-details').show();
```
Comment (sloria) [inline website/static/js/projectorganizer.js]: This jQuery object should be cached to be avoid multiple DOM traversals; this selector in particular is used throughout this module.

**Comment by caneruguz** [comment]
I added the caching, especially where the comment was and looked through the document. many of the other instances create unique divs so there isn't much opportunity to cache there and I didn't want to do a global caching of project-details, because we are dynamically changing the state of these divs so I think it's best to keep the caching within the function. 

I looked through fangorn.js as well for this but didn't find any cache appropriate place there.

File: website/static/js/projectorganizer.js [hunk 3]
```diff
@@ -741,7 +742,8 @@ function _poMultiselect(event, tree) {
     var tb = this,
         selectedRows = filterRowsNotInParent.call(tb, tb.multiselected),
         someItemsAreFolders,
-        pointerIds;
+        pointerIds,
+        detailDiv = $('.project-details');
```
Comment (sloria) [inline website/static/js/projectorganizer.js]: > ll variables that are used to store/cache jQuery objects should have a name prefixed with a $. 

http://lab.abhinayrathore.com/jquery-standards/#Variables via cosdev.readthedocs.org/en/latest/style_guides/javascript.html

**Comment by sloria** [comment]
$('.project-details') can safely be cached, since the DOM element is not dynamically created or removed. Did this during merge.

File: website/static/js/projectorganizer.js (hunk 3 above)
Comment (sloria) [inline website/static/js/projectorganizer.js]: Fixed during merge.

**Pull request merged**